lightwood/encoders/time_series/helpers/test_ts.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import torch
+
+
+def test_ts(encoder, queries, answers, params):
```

Comment:
Please make it a unit test under tests/unit_tests, or maybe even multiple unit tests